tasks/sds.py

```diff
@@ -0,0 +1,38 @@
+import os
+import sys
+import tempfile
+
+from invoke import task
+
+from tasks.rtloader import get_dev_path
+
+is_windows = sys.platform == "win32"
+is_darwin = sys.platform == "darwin"
+
+
+@task
+def build_library(ctx):
+    """
+    Build the SDS shared library
+    """
+    if is_windows:
+        print("not supported")
+        return
+    with tempfile.TemporaryDirectory() as temp_dir:
+        with ctx.cd(temp_dir):
+            ctx.run("git clone https://github.com/DataDog/dd-sensitive-data-scanner")
+            with ctx.cd("dd-sensitive-data-scanner/sds-go/rust"):
+                ctx.run("cargo build --release")
```

Comment:
Not familiar with this repo but would it make sense to checkout the `042de62f5a24fbceb4f4849256c3ee5c005b7057` commit? To use the same version the Agent does

Reply:
What about adding this to the build image? Should we build the lib on `https://github.com/DataDog/dd-sensitive-data-scanner` and then install it on build images? The risk of this is to get rate limited by github.

Reply:
> would it make sense to checkout the ...

It absolutely does, I should've mention it in the PR description sorry: the library has no versioning strategy for now but will in the coming weeks. I plan to adapt this change in a separate PR to checkout a given version of the lib (which might be configured in release.json or similar), when it's something that'll be doable.

> What about adding this to the build image? [...] and then install it on build images? 

It would need a build image PR/rebuild every time we want to update the library version, which is not ideal, but that we can still consider if we're really afraid of being rate limited by github for this.

Reply:
Thanks!

Reply:
> if we're really afraid of being rate limited by github for this.
We are, it's being a nightmare. 

We have guidelines to reduce external dependencies from runners, eventually CI Reliability will cut all external access (eventually).

Reply:
Discussed offline:
  * using a version for the SDS library will be added soon
  * git clones are a not a problem